test/unit/spec/common/jobCommChannel-spec.js

```diff
@@ -243,19 +447,26 @@ define([
                     .initCommChannel()
                     .then(() => {
                         expect(comm.comm).not.toBeNull();
+                        spyOn(comm, 'sendCommMessage').and.callFake((...args) => {
+                            return comm.sendCommMessage.and.originalFn.call(comm, ...args);
+                        });
```

Comment:
add in an extra spy -- this was mostly for symmetry with the previous tests involving the stored messages